<ui version="4.0" >
 <class>First</class>
 <widget class="QMainWindow" name="First" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>First</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QRadioButton" name="radioButton" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>90</y>
      <width>102</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>RadioButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>40</y>
      <width>102</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>PushButton3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>20</y>
      <width>102</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>PushButton2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>110</y>
      <width>102</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>340</y>
      <width>84</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text" >
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>60</y>
      <width>102</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>PushButton4</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>0</y>
      <width>441</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex" >
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab" >
     <attribute name="title" >
      <string>Tree</string>
     </attribute>
     <widget class="QTreeView" name="treeView" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>10</y>
        <width>421</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3" >
     <attribute name="title" >
      <string>Table</string>
     </attribute>
     <widget class="QListWidget" name="listWidget" >
      <property name="geometry" >
       <rect>
        <x>40</x>
        <y>0</y>
        <width>120</width>
        <height>94</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2" >
     <attribute name="title" >
      <string>Calendar</string>
     </attribute>
     <widget class="QCalendarWidget" name="calendarWidget" >
      <property name="geometry" >
       <rect>
        <x>30</x>
        <y>10</y>
        <width>208</width>
        <height>175</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QToolBox" name="toolBox" >
    <property name="geometry" >
     <rect>
      <x>540</x>
      <y>30</y>
      <width>201</width>
      <height>189</height>
     </rect>
    </property>
    <property name="currentIndex" >
     <number>1</number>
    </property>
    <widget class="QWidget" name="page" >
     <property name="geometry" >
      <rect>
       <x>0</x>
       <y>0</y>
       <width>96</width>
       <height>26</height>
      </rect>
     </property>
     <attribute name="label" >
      <string>Time</string>
     </attribute>
     <widget class="QTimeEdit" name="timeEdit" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>10</y>
        <width>118</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2" >
     <property name="geometry" >
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>90</height>
      </rect>
     </property>
     <attribute name="label" >
      <string>Date</string>
     </attribute>
     <widget class="QDateEdit" name="dateEdit" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>10</y>
        <width>110</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3" >
     <property name="geometry" >
      <rect>
       <x>0</x>
       <y>0</y>
       <width>96</width>
       <height>26</height>
      </rect>
     </property>
     <attribute name="label" >
      <string>Time/Date</string>
     </attribute>
     <widget class="QDateTimeEdit" name="dateTimeEdit" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>20</y>
        <width>194</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="PushButton_1" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>102</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>PushButton1</string>
    </property>
    <property name="flat" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>140</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="standardButtons" >
     <set>QDialogButtonBox::Ok</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>614</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionOpen" />
    <addaction name="actionClose" />
   </widget>
   <widget class="QMenu" name="menuOptions" >
    <property name="title" >
     <string>Options</string>
    </property>
    <addaction name="actionAction1" />
    <addaction name="actionAction2" />
    <addaction name="actionFitter" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuOptions" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionAction1" />
   <addaction name="separator" />
   <addaction name="actionAction2" />
   <addaction name="separator" />
   <addaction name="actionOpen" />
   <addaction name="separator" />
   <addaction name="actionClose" />
   <addaction name="separator" />
   <addaction name="actionFitter" />
  </widget>
  <action name="actionAction1" >
   <property name="icon" >
    <iconset resource="First.qrc" >:/images/images/color.png</iconset>
   </property>
   <property name="text" >
    <string>Action1</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="icon" >
    <iconset resource="First.qrc" >:/images/images/open.png</iconset>
   </property>
   <property name="text" >
    <string>Open...</string>
   </property>
  </action>
  <action name="actionClose" >
   <property name="icon" >
    <iconset resource="First.qrc" >:/images/images/close.png</iconset>
   </property>
   <property name="text" >
    <string>Close...</string>
   </property>
  </action>
  <action name="actionAction2" >
   <property name="icon" >
    <iconset resource="First.qrc" >:/images/images/dynlist.png</iconset>
   </property>
   <property name="text" >
    <string>Action2</string>
   </property>
  </action>
  <action name="actionFitter" >
   <property name="icon" >
    <iconset resource="First.qrc" >:/images/images/fitter.png</iconset>
   </property>
   <property name="text" >
    <string>Fitter</string>
   </property>
  </action>
  <action name="actionAction3" >
   <property name="text" >
    <string>Action3</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="First.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>PushButton_1</sender>
   <signal>released()</signal>
   <receiver>radioButton</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>60</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel" >
     <x>60</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitButton</sender>
   <signal>released()</signal>
   <receiver>First</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>215</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel" >
     <x>377</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
